oss-internship-2020/libtiff/sandboxed.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <linux/futex.h>
+#include <syscall.h>
+
+#include <optional>
+#include <utility>
+
+#include "tiff_sapi.sapi.h"  // NOLINT(build/include)
+
+namespace {
+
+class TiffSapiSandbox : public TiffSandbox {
+ public:
+  TiffSapiSandbox(std::optional<std::string> file = std::nullopt,
+                  std::optional<std::string> dir = std::nullopt)
+      : file_(std::move(file)), dir_(std::move(dir)) {}
+
+ private:
+  std::unique_ptr<sandbox2::Policy> ModifyPolicy(
+      sandbox2::PolicyBuilder*) override {
+    auto builder = std::make_unique<sandbox2::PolicyBuilder>();
```

Comment:
Seems there is no need in this function to dynamically allocate `PolicyBuilder`.
It can be safely stack-allocated.